Gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var babel  = require('gulp-babel');
+var eslint = require('gulp-eslint');
+var gulp   = require('gulp');
+var rimraf = require('rimraf');
+
+gulp.task('clean', function (cb) {

```

Comment:
add an additional empty line between `vars` block and `tasks` block. We usually add doubled space between logical blocks of code. It's not a strict rule but try do it